src/components/AppContainer.jsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import SideBar from './SideBar-component/SideBar';
+import AppMainContent from './MainContent-component/AppMainContent';
+import { ThemeContext } from '../context';
+
+const StyledAppContainer = styled.div`
+  width: 90%;
+  max-width: 90%;
+  min-width: 90%;
+  height: 90vh;
+  display: flex;
+  margin: 0 auto;
+  background-color: #eaeff1;
+  border-radius: 15px;
+
+  @media screen and (max-width: 768px) {
+    width: 100%;
+    max-width: 100%;
+    min-width: 100%;
+    padding: 0 5%;
+    background-color: #bcdbad;
+    height: 100dvh;
+  }
+`;
+
+export default function AppContainer() {
+  const [selectedDayString, setSelectedDayString] = useState(
+    new Date().toISOString().slice(0, 10)
+  );
+
+  const [selectedDayTodoList, setSelectedDayTodoList] = useState(null);
+  const [selectedDayDoneList, setSelectedDayDoneList] = useState(null);
+
+  useEffect(() => {
+    const todo =
+      localStorage.getItem(`${selectedDayString}todo`) === null
+        ? []
+        : JSON.parse(localStorage.getItem(`${selectedDayString}todo`));
+
+    const done =
+      localStorage.getItem(`${selectedDayString}done`) === null
+        ? []
+        : JSON.parse(localStorage.getItem(`${selectedDayString}done`));
+    setSelectedDayTodoList(todo);
+    setSelectedDayDoneList(done);
+  }, []);
```

Comment:
useEffect 훅이 실행될 때 의존성 배열을 빈 배열로 설정하여 컴포넌트가 마운트될 때만 실행되도록 하면 selectedDayString 상태가 변경될 때 localStorage에서 해당 날짜의 할 일과 완료된 일을 다시 불러와야 해서 의존성 배열에 selectedDayString을 포함시키면 electedDayString이 변경될 때마다 useEffect가 실행되어 해당 날짜의 할 일과 완료된 일을 localStorage에서 불러와서 불필요한 동작이 감소할 것 같아요!

```suggestion
useEffect(() => {
  const todo = JSON.parse(localStorage.getItem(`${selectedDayString}todo`)) || [];
  const done = JSON.parse(localStorage.getItem(`${selectedDayString}done`)) || [];
  setSelectedDayTodoList(todo);
  setSelectedDayDoneList(done);
}, [selectedDayString]);
```